### PR TITLE
Add time management guide for mentors to use for demo

### PR DESCRIPTION
**Pull request by veredrec**
Relates #628 #592

After discussing the benefits of having a demo for the students to introduce pomodoro clock and other time management tools, this document gives several examples for tools (including links and screenshots).

### CODE REVIEW

**Comment by rinoma** [comment]
When would be the best time to show this? First week on the morning of the projects(day 4)?

**Comment by Jen-Harris** [comment]
Yes @rinoma I agree 👍

File: coursebook/general/pomodoro-guide.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+## Time Management Tools
+
+##### We would like to introduce pomodoro clock and other time management tools to the students as a strategy to use when working on projects.
+##### The mentors will explain the benefits of using time management tools and will demo several example.
```
Comment (shiryz) [inline coursebook/general/pomodoro-guide.md]: *exmaples, typo

Comment (veredrec) [reply coursebook/general/pomodoro-guide.md]: Thanks @shiryz I fixed it :+1:

**Pull request merged**